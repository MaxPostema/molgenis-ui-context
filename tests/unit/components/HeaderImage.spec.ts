import { shallowMount } from '@vue/test-utils'
import HeaderImage from '../../../src/components/HeaderImage.vue'

describe('HeaderImage.vue', () => {
  it('should match the snapshot', () => {
    const wrapper = shallowMount(HeaderImage, {
      propsData: {
        imgSrc: 'https://my/image.png',
        imgMaxHeight: 200

      }
    })
    expect(wrapper.html()).toMatchSnapshot()
  })
})
