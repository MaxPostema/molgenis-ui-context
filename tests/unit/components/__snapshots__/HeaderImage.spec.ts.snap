// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderImage.vue should match the snapshot 1`] = `<div id="top-logo-vue-banner" class="bg-light" style="height: 200px;"><span class="mg-top-logo-helper"></span> <a href="/"><img id="logo-top" alt="" src="https://my/image.png" class="logo-top" style="max-height: 200px;"></a></div>`;
